.harness/cichecks.yaml

```diff
@@ -36,7 +36,7 @@ pipeline:
                         identifier: run_lint
                         spec:
                           connectorRef: account.dockerhub_datarobot_read
-                          image: python:3.9
```

Comment:
I planned to move to 3.12 later: https://datarobot.atlassian.net/browse/MLDEV-1135
But I would run tests for all supported pythons starting 3.9 https://datarobot.atlassian.net/browse/MLDEV-1134